### ISSUE TITLE
Multiple azure functions into the same App Service account

### ISSUE THREAD

**Issue by marcodali** [microsoft/vscode-azurefunctions#4129]
Using the "Create Function App in Azure... (Advanced)" option how can I tell the wizard not to create a new App Service account but to use instead a previously created one.

So what I want is a relationship of one App Service account to many Azure functions.

How can I instruct this extension to do so??

**Comment by nturinski** [comment]
You should be able to select which `App Service account` on this prompt.
![image](https://github.com/microsoft/vscode-azurefunctions/assets/5290572/65fa8dee-15c4-426e-8f2c-0140848c7452)

In order to see this prompt, you have to had clicked `App Service Plan` or `Premium` when selecting a hosting plan
![image](https://github.com/microsoft/vscode-azurefunctions/assets/5290572/5b8ea8df-bd92-4042-9371-38679c768537)

**Comment by nturinski** [comment]
Please let me know if this resolves your issue.

**Comment by marcodali** [comment]
Yep I’ll close it    ---- On Wed, 15 May 2024 11:52:56 -0600  ***@***.******@***.***> wrote ----  
Please let me know if this resolves your issue.
